### PR TITLE
Modify test files based on local sdk and stakeholder

### PR DESCRIPTION
**Pull request by MSNTCS**

### CODE REVIEW

File: test/src/stakeholder/src/rpc.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import { SDK } from "codechain-sdk";
-import { PlatformAddress } from "codechain-sdk/lib/core/classes";
+// @ts-ignore
```
Comment (majecty) [inline test/src/stakeholder/src/rpc.ts]: Why did you add this line?

**Comment by MSNTCS** [comment]
@majecty,  the failed test was run perfectly in my machine and also was run without any problem here https://travis-ci.com/MSNTCS/foundry.
